styles/index.styles.ts

```diff
@@ -68,4 +70,26 @@ export const AppWrapper = styled.div`
 export const Content = styled.div`
   margin: 30px;
   display: flex;
+  flex-wrap: wrap;
```

Comment:
This could also be done by defining flex-direction: row for desktop and flex-direction: column for phone. Just saying because u will need it in the future :)